contracts/Overtime/SportsAMMV2.sol

```diff
@@ -453,24 +505,33 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
 
     function _trade(
         ISportsAMMV2.TradeData[] memory _tradeData,
-        uint _buyInAmount,
-        uint _expectedPayout,
-        uint _additionalSlippage,
-        address _differentRecipient,
-        bool _sendDefaultCollateral
+        TradeDataInternal memory tradeDataInternal,
+        address requester
```

Comment:
I would add the `requester` in the `tradeDataInternal` struct. 
It will leave more space in the stack

Reply:
added